ompi/mca/coll/base/coll_tags.h

```diff
@@ -52,8 +52,9 @@
 #define MCA_COLL_BASE_TAG_SCATTERV -26
 #define MCA_COLL_BASE_TAG_BLOCKING_END -26
 
```

Comment:
Please do not reorder the tags, add yours at the end of the list.

Reply:
Hi, thanks for comment. I've fixed: i put the UCC tag right before STATIC_END defining STATIC_END = TAG_UCC - 1. 
This is because UCC only needs 1 tag value and tags that go after they take entire ranges. Will this work?